packages/editor/src/components/rich-text/index.native.js

```diff
@@ -69,25 +84,68 @@ export class RichText extends Component {
 		return true;
 	}
 
+	isFormatActive( format ) {
+		return this.state.formats[ format ] && this.state.formats[ format ].isActive;
```

Comment:
You can use `lodash.get` to simplify this code:
```js
return get( this.state, [ 'formats', format, 'isActive' ], false );
```

Reply:
I think this was copy paste from the original code at the time.

Reply:
Another reason, to split this file into two chunks :)